redis_sentinel/check.py

```diff
@@ -0,0 +1,212 @@
+            }]
+        """
+        sentinels_stats = redis_conn.sentinel_sentinels(master_name)
+        # sentinel_stats returns stats for other sentinels only
+        # so increment once for current sentinel
+        self.increment('redis.sentinel.ok_sentinels', tags=master_tags)
+        for stats in sentinels_stats:
+            sentinel_tags = ['sentinel_ip:%s' % stats['ip']] + base_tags
+            if stats['is_odown'] or stats['is_sdown']:  # sentinel keeps track of old sentinels
+                continue
+            self.increment('redis.sentinel.ok_sentinels', tags=master_tags)
+            self.gauge(
+                'redis.sentinel.pending_commands',
+                stats['pending-commands'], tags=['sentinel'] + sentinel_tags
+            )
+            self.gauge(
+                'redis.sentinel.ping_latency',
+                stats['last-ping-reply'] - stats['last-ping-sent'],
+                sentinel_tags
+            )
+            self.gauge(
+                'redis.sentinel.last_ok_ping_latency',
+                stats['last-ping-reply'] - stats['last-ok-ping-reply'],
+                sentinel_tags
+            )
+
+    def _process_slaves_stats(self, redis_conn, master_name, base_tags, master_tags):
+        """
+            [{
+                'down-after-milliseconds': 5000,
+                'flags': 'slave',
+                'info-refresh': 2628,
+                'ip': '10.1.2.3',
+                'is_disconnected': False,
+                'is_master': False,
+                'is_master_down': False,
+                'is_odown': False,
+                'is_sdown': False,
+                'is_sentinel': False,
+                'is_slave': True
+                'last-ok-ping-reply': 429,
+                'last-ping-reply': 429,
+                'last-ping-sent': 0,
+                'master-host': '10.1.2.3',
+                'master-link-down-time': 0,
+                'master-link-status': 'ok',
+                'master-port': 6379,
+                'name': '10.1.2.3:6379',
+                'pending-commands': 0,
+                'port': 6379,
+                'role-reported': 'slave',
+                'role-reported-time': 3124725,
+                'runid': '123456789abcdef',
+                'slave-priority': 100,
+                'slave-repl-offset': 12345678,
+            }]
+        """
+        slaves_stats = redis_conn.sentinel_slaves(master_name)
+        for stats in slaves_stats:
+            if stats['is_odown'] or stats['is_sdown']:  # sentinel keeps track of old slaves
+                continue
+            self.increment('redis.sentinel.ok_slaves', tags=master_tags)
+            slave_tags = ['slave_ip:%s' % stats['ip']] + base_tags
+            self.gauge(
+                'redis.sentinel.pending_commands', stats['pending-commands'],
+                tags=['slave'] + slave_tags
+            )
+
+            self.service_check(
+                'redis.sentinel.slave_is_disconnected',
```

Comment:
nitpick: we seem to consistently use affirmative statements for service check names (i.e. `slave_connected` or `slave_connection_status` instead of `slave_is_disconnected`), not sure it'd make sense here though.
Disregard if you disagree